### DISCUSSION TITLE
What are SOPS?

### DISCUSSION THREAD

**Discussion by buschNT** [unikubehq/frontend#117] [General]
Each unikube project has a SOPS section, which might be hard to understand for some people (me included).
Maybe this is clear for everyone, but I was not 100% sure what it means and google showed the following, which was not helping ;) :

search "SOPS" -> wikipedia

> A standard operating procedure (SOP) is a set of step-by-step instructions compiled by an organization to help workers carry out routine operations. SOPs aim to achieve efficiency, quality output and uniformity of performance, while reducing miscommunication and failure to comply with industry regulations.

Just by searching "SOPS kubernetes" I found https://fluxcd.io/docs/guides/mozilla-sops/

Thus, I just wanted to start a discussion here to see/get feedback if SOPS is commonly known by the target group of unikube? For me it looks like kubernetes specific knowledge.

**Comment by SteinRobert** [comment]
It is definitely not common knowledge what SOPS actually are. However we'd like to nudge the users a bit to make use of them, since we believe that this makes the applications more secure.

I agree that an explanation would be really helpful.

Idea: We use some kind of "help" icon for those terms (since "Deck" itself is a Unikube concept which has to be explained as well) which provides the user with a tooltip. The tooltip includes a short remark (1 sentence) and a link to a more detailed explanation (preferably in the unikube docs knowledge base).

What do you think?
